src/parser/shared/parserGenerator.ts

```diff
@@ -19,7 +19,10 @@ export interface DXFParserSnippet {
     // 만약 Abort 심볼이 반환될 경우 값에 대입하지 않고 읽은 것을 한 칸 되돌리고
     // 종료함
     parser?(curr: ScannerGroup, scanner: DxfArrayScanner, entity: any): any;
-    isMultiple?: boolean; // code가 여러 번 들어올 수 있는 경우, true로 표기
+    /** When specific group code can be read multiple times, set this `true` */
+    isMultiple?: boolean;
+    /** When isMultiple is `true`, save array when `false`, replace as is when `true` */
+    isReducible?: boolean; 
```

Comment:
JSDoc type comment will be helpful. Also I translated legacy Korean comment into English.